train_vnext/training.py

```diff
@@ -158,17 +142,85 @@ def process_post_training_csv(csv_path, training_id, classification_name_to_clas
                 prediction_label = PredictionLabel(training_id, 
                                     int(row[0].split('.')[0]), 
                                     classification_name_to_class_id[class_name], 
-                                    float(row[2]), 
-                                    float(row[3]), 
-                                    float(row[4]), 
-                                    float(row[5]), 
+                                    float(row[2])*float(row[6]), 
```

Comment:
@samiyaakhtar row[6] is Height and row[7] is Width so we what the XMin and XMax to be divided by the width (x-axis) and YMin YMax to be divided by the Height (Y Axis)

Reply:
good catch

Reply:
@samiyaakhtar sorry I gave instructions on how to convert from absolute to relative. You're attempting to convert from relative to absolute so multiplication is correct instead of divide.

Perhaps we should have static methods on the Prediction Label that do this conversion for us. Int would be more declarative to have: 

```
prediction_label = PredictionLabel(...)
prediction_label = prediction_label.ConvertToAbsolute()
```
What do you think?

Reply:
yeah i think this is a better idea, let me do this as part of this PR later today

Reply:
I thought it was better to place the convert to relative in the Tag class since we convert human predicted labels to relative. Let me know if you have more feedback.